src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -234,8 +233,8 @@
       SourceBuildMetadataDir="$(TarballGitInfoDir)"
       Dependencies="@(SourceBuildRepos)" />
 
-    <Message Text="--> Dependencies for $(TarballVersionDetailsFile):" Importance="High" />
-    <Message Text="-->     [@(SourceBuildRepos)]" Importance="High" />
+    <Message Text="--> Dependencies for $(TarballVersionDetailsFile):" Importance="High" Condition=" '@(SourceBuildRepos)' != '' " />
+    <Message Text="-->     %(SourceBuildRepos.SourceBuildRepoName) / %(SourceBuildRepos.Sha)" Importance="High" Condition=" '@(SourceBuildRepos)' != '' " />
```

Comment:
Maybe `@(SourceBuildRepos->'%(SourceBuildRepoName) / %(Sha)')` would look better?

Reply:
That gives me this:
```
   -->     runtime / f52d8c59bb49360eb2cbeeb863c5856ebd62adda;aspnetcore / 0438e7ec04808230c67b00caa3d584bd2b991f57;test-templates / a47740f5f3936da8dcebe9e8868b6311bc4402f8;sdk / 80a77fd427d34e038ff2fa21f6c807695292cf7e;fsharp / ee745b0455fe651b6f5e5f9b85fc59a7b2574395;vstest / ef0aa102e9c7d5cf55af06c2747d11e07d12b69a;linker / 6252a2194dd32911db2c0669fc818555687d5570;roslyn / 65dc2845b5287ac972f7299616c13d3270720e44;msbuild / 50f6081beda4c565252ff51be4ab1fe27005372b;nuget-client / 26a0cf27a2d4dcf52d19238606c269d6fed9ff66;xdt / 698fdad58fa64a55f16cd9562c90224cc498ed02;deployment-tools / c3ad00ae84489071080a606f6a8e43c9a91a5cc2;diagnostics / ab3eb7a525e31dc6fb4d9cc0b7154fa2be58dac1;source-build-externals / ee790b0477953dd30ea83e0e47af909a04fd0ca3;symreader / 7b9791daa3a3477eb22ec805946c9fff8b42d8ca;arcade / 6a638cd0c13962ab2a1943cb1c878be5a41dd82e;source-build-reference-packages / c1bc837421e41a673cb30bf670c0cdd83ac09b75;sourcelink / 508c31bb736f3e6d69b8f47fa9ee38eedd9ce785;xliff-tasks / 8dfdebfefc8acef481f28edadbf7f75a8d6efde4
```

whereas the current one this:
```
   -->     runtime / f52d8c59bb49360eb2cbeeb863c5856ebd62adda
   -->     aspnetcore / 0438e7ec04808230c67b00caa3d584bd2b991f57
   -->     test-templates / a47740f5f3936da8dcebe9e8868b6311bc4402f8
   -->     sdk / 80a77fd427d34e038ff2fa21f6c807695292cf7e
   -->     fsharp / ee745b0455fe651b6f5e5f9b85fc59a7b2574395
   -->     vstest / ef0aa102e9c7d5cf55af06c2747d11e07d12b69a
   -->     linker / 6252a2194dd32911db2c0669fc818555687d5570
   -->     roslyn / 65dc2845b5287ac972f7299616c13d3270720e44
   -->     msbuild / 50f6081beda4c565252ff51be4ab1fe27005372b
   -->     nuget-client / 26a0cf27a2d4dcf52d19238606c269d6fed9ff66
   -->     xdt / 698fdad58fa64a55f16cd9562c90224cc498ed02
   -->     deployment-tools / c3ad00ae84489071080a606f6a8e43c9a91a5cc2
   -->     diagnostics / ab3eb7a525e31dc6fb4d9cc0b7154fa2be58dac1
   -->     source-build-externals / ee790b0477953dd30ea83e0e47af909a04fd0ca3
   -->     symreader / 7b9791daa3a3477eb22ec805946c9fff8b42d8ca
   -->     arcade / 6a638cd0c13962ab2a1943cb1c878be5a41dd82e
   -->     source-build-reference-packages / c1bc837421e41a673cb30bf670c0cdd83ac09b75
   -->     sourcelink / 508c31bb736f3e6d69b8f47fa9ee38eedd9ce785
   -->     xliff-tasks / 8dfdebfefc8acef481f28edadbf7f75a8d6efde4
```

I guess your solution repeats everything behind the `->` and mine repeats the whole `<Message>` task